src/main/java/org/apache/maven/shared/dependency/analyzer/asm/DependencyClassFileVisitor.java

```diff
@@ -75,6 +75,9 @@ public void visitClass(String className, InputStream in) {
             // some bug inside ASM causes an IOB exception. Log it and move on?
             // this happens when the class isn't valid.
             logger.warn("Unable to process: " + className);
```

Comment:
Might be helpful to include the exception in the warning here as well

Reply:
@elharo 
I update the call.